lib/core/presentation/widgets/event/rsvp_event_success_popup_widget.dart

```diff
@@ -91,36 +92,47 @@ class RSVPEventSuccessPopupPage extends StatelessWidget {
               top: 0,
               left: 0,
               right: 0,
-              child: Padding(
-                padding: EdgeInsets.symmetric(horizontal: Spacing.smMedium),
-                child: Row(
-                  mainAxisSize: MainAxisSize.max,
-                  mainAxisAlignment: MainAxisAlignment.spaceBetween,
-                  crossAxisAlignment: CrossAxisAlignment.center,
-                  children: [
-                    const Spacer(),
-                    Container(
-                      width: 125.w,
-                      height: Sizing.medium,
-                      decoration: BoxDecoration(
-                        color: colorScheme.onPrimary.withOpacity(0.09),
-                        borderRadius: BorderRadius.circular(LemonRadius.normal),
-                      ),
-                      child: Row(
-                        mainAxisAlignment: MainAxisAlignment.center,
-                        children: [
-                          Assets.icons.icUserAdd.svg(),
-                          SizedBox(width: Spacing.extraSmall),
-                          Text(
-                            t.event.inviteFriends,
-                            style: Typo.small.copyWith(
-                              color: colorScheme.onSecondary,
-                            ),
-                          )
-                        ],
-                      ),
-                    )
-                  ],
+              child: InkWell(
+                onTap: () => showDialog(
+                  context: context,
+                  builder: (context) => ComingSoonModal(
```

Comment:
We have utils ` showComingSoonDialog(context)`. Use this instead

Reply:
👍 almost forgot about this . Thank you